saleor/core/utils/random_data.py

```diff
@@ -767,7 +756,13 @@ def create_fulfillments(order):
 
 
 def create_fake_order(discounts, max_order_lines=5, create_preorder_lines=False):
-    channel = Channel.objects.all().order_by("?").first()
+    channel = (
+        Channel.objects.filter(
+            Q(slug=settings.DEFAULT_CHANNEL_SLUG) | Q(slug="channel-pln")
```

Comment:
I think that you could simplify this query as you can handle it without Q object:
```
Channel.objects.filter(slug__in=[settings.DEFAULT_CHANNEL_SLUG, "channel-pln"]).order_by("?")
         .first()
```